static/js/typeahead_helper.js

```diff
@@ -66,6 +66,21 @@ exports.render_person = function (person) {
     return person.full_name + " <" + person.email + ">";
 };
 
+exports.render_stream = function (token, stream) {
+    var desc = stream.description;
+    var short_desc = desc.substring(0, 35);
+
+    if (desc === short_desc) {
+        desc = exports.highlight_with_escaping(token, desc);
+    } else {
+        desc = exports.highlight_with_escaping(token, short_desc) + "...";
+    }
+
+    var name = exports.highlight_with_escaping(token, stream.name);
+
+    return name + '&nbsp;&nbsp;<small style = "opacity: 0.6;">' + desc + '</small>';
```

Comment:
I wasn't entirely sure whether `style = "opacity: 0.6;"` should be inline or a class in `zulip.css`.